Archisinal.md

```diff
@@ -0,0 +1,310 @@
+
+Currently serving as the Special Projects Lead at Parity Technologies, Tom has successfully transitioned his career from the construction industry to the world of Web3. His background in both traditional industries and emerging technologies, along with his adaptability and ability to excel in diverse environments, make him a valuable asset for spearheading innovation in a large-market sector which is yet to fully embrace the benefits of blockchain technology.
+
+**Danylo Semirazov -** blockchain developer. 9+ years in IT, worked in financial, networking and security domains. 3+ yeas in web3. Master degree in Information Security. Contributions to Polkadot ecosystem: core developer of [OpenBrush contracts](https://github.com/Supercolony-net/openbrush-contracts) - ink! smart contracts library, CTO at [Asylum space](https://github.com/w3f/Grants-Program/blob/master/applications/asylum.md) - metaverse [NFT standards](https://docs.asylum.space/) (supported by W3F Grants Program), participated in developing DEX and lending platform in ink! from scratch, contributed to substrate and ink! repositories (upgradable contracts), speaker at Polkadot Decoded and WASM conference. Co-founder of [Evolute](https://docsend.com/view/s/ckc9xuhhmmwpwd8k) - web3 game studio. Tech stack: Rust, Substrate, Solidity, C#, Java, JS, web3.js, polkadot.js, Typescript, React.js, Node.js
+
+**Oleh Kulykovskyi** - front-end developer with 4.5 years of work experience including 3.5 years working with React & Next.JS, mostly in TypeScript language. Proficient with numerous business-grade tools like Redux, GraphQL, Git, Github, animation libraries (GSAP, Framer), MUI, Jest etc. Implemented a broad range of web solutions from landing pages focused on visuals to complex system of business-grade interconnected apps. Created and implemented a web3 platform for a project in Polkadot ecosystem - [Asylum space](https://github.com/w3f/Grants-Program/blob/master/applications/asylum.md). Developed an npm package as a solution for integrating blockchain into React. Tech stack: JS, React.js, Next.js, web3.js, polkadot.js, Typescript.
+
+**Artem Smolyarenko** - UI/UX designer with over 3 years experience. He specializes in brand development in all aspects and have an excellent understanding of typography, color psychology, as well as grid, web and mobile principles. Artem worked on different types of products with startups and corporate teams from around the world. Skills: UI/UX design, product design, web and mobile app design. Tech stack & tools: Figma, Adobe Photoshop, Adobe Illustrator, Adobe XD.
+
+### Team Code Repos
+
+- **Archisinal** [https://github.com/Archisinal](https://github.com/Archisinal)
+- **Danylo Semirazov** - [github.com/sventime](https://github.com/sventime), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+- **Oleh Kulykovskyi**  - [github.com/blindbeat](http://github.com/blindbeat), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
```

Comment:
There don't seem to be a lot of contributions on GitHub for either team member. Would you be able to point to some, specially related to Rust/Substrate/Solidity?

Reply:
PR updated with contributions of team members. cc: @sventime